apps/allocations/app/components/App/Budgets.js

```diff
@@ -0,0 +1,63 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+
+import { Viewport } from '@aragon/ui'
+
+import { CARD_STRETCH_BREAKPOINT } from '../../utils/responsive'
+import { Budget } from '../Card'
+
+const Budgets = ({
+  budgets,
+  onNewAllocation,
+  onEdit,
+  onDeactivate,
+  onReactivate,
+}) => {
+  if (!budgets) return
+
+  return (
+    <Viewport>
+      {({ width }) => {
+        const screenSize = width
+
+        return (
+          <StyledAccounts screenSize={screenSize}>
+            {budgets.map(({ budgetId, data }) => (
+              <Budget
+                key={budgetId}
+                id={budgetId}
+                name={data.name}
+                amount={data.amount}
+                currency={data.currency}
```

Comment:
I personally think `token` is more appropriate for the space we are working on, but let's discuss this better to achieve a consensus before using one or another everywhere.

Reply:
I don't have any objections for this. Please feel free to update accordingly.